homeassistant/components/octoprint/sensor.py

```diff
@@ -123,14 +123,14 @@ def native_value(self):
         """Return sensor state."""
         printer: OctoprintPrinterInfo = self.coordinator.data["printer"]
         if not printer:
-            return None
+            return "Disconnected"
```

Comment:
This is not correct. If the printer is not connection, this sensor should go unavailable right?

Reply:
No. Please see comment in your requested change below.